composer.json

```diff
@@ -17,7 +17,7 @@
         "php": "^5.3.6 || >=7.0 <7.2",
         "ext-tokenizer": "*",
         "sebastian/diff": "^1.1",
-        "symfony/console": "^2.3 || ^3.0",
+        "symfony/console": "^2.4 || ^3.0",
```

Comment:
As @SpacePossum said, the class is only used for tests and is not very complex, maybe we can just write our own?